src/components/Header.js

```diff
@@ -2,10 +2,19 @@ import React from 'react'
 
 const Header = () => {
     return (
-        <diV className='header'>
-            <h1>Social Warming</h1>
-            <p>Promo text</p>
-        </diV>
+        <div className='header'>
+            <div class="title"> 
+                <h1 class="firstTitle">Social</h1>
+                <h2 class="secondTitle">Warming</h2>
+            </div>
+            <p class="text">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+            <br></br>
+            <br></br>
```

Comment:
Is dit nodig?

Reply:
yes, had dit als alternative gebruikt van margin bij de button

Reply:
Kan dit dan niet beter gewoon in de css?